manifests/reposync-crd.yaml

```diff
@@ -1186,8 +1186,52 @@ spec:
                     type: object
                   values:
                     description: values to use instead of default values that accompany
-                      the chart
+                      the chart. Format values the same as default values.yaml. These
+                      values will take precedence if used in conjunction with valuesFrom.
+                      How to handle multiple valuesFiles is determined by `valuesKeyMergeMode`.
                     x-kubernetes-preserve-unknown-fields: true
+                  valuesFrom:
+                    description: valuesFrom holds references to objects in the cluster
+                      that represent values to use instead of default values that
+                      accompany the chart. Currently, only ConfigMaps are supported.
+                      Objects listed later will take precedence. How to handle multiple
+                      valuesFiles is determined by `valuesKeyMergeMode`.
+                    items:
+                      description: ValuesFrom holds references to objects in the cluster
+                        that represent values to use instead of default values that
+                        accompany the chart.
+                      properties:
+                        key:
+                          default: values.yaml
+                          description: key represents the object data key to read
+                            the value from.
+                          type: string
+                        kind:
+                          description: kind represents the Object kind. Must be `ConfigMap`.
+                            Required.
+                          enum:
+                          - ConfigMap
+                          type: string
+                        name:
+                          description: name represents the Object name. Required.
+                          type: string
+                      type: object
+                    type: array
+                  valuesKeyMergeMode:
```

Comment:
"valuesFileMergeStrategy" might be preferable

Reply:
I'm happy to update the flag name in helm-sync, but we had a really long discussion and a lot of back and forth on this field name (and its options), and I'd prefer not to open that up again